src/Core/Services/UsernameGenerationService.cs

```diff
@@ -12,25 +12,26 @@ namespace Bit.Core.Services
 {
     public class UsernameGenerationService : IUsernameGenerationService
     {
-        private const string DefaultGenerated = "-";
+        private const string DEFAULT_GENERATED = "-";
```

Comment:
Put this in `Bit.Core.Constants` so that it can be reused here and on the `GeneratorPageViewModel `